### PR TITLE
remove redux logger if production

### PR DESCRIPTION
**Pull request by bolicd**
Redux logger middleware is not show if production

### CODE REVIEW

**Comment by mz026** [comment]
How about making the [`predicate` option](https://github.com/mz026/universal-redux-template/blob/master/app/store/configureStore.js#L12) return `false` if in production mode?

**Comment by bolicd** [comment]
@mz026 Hey. 

According to official docs:
predicate, // If specified this function will be called before each action is processed with this middleware.

so you are still initializing logger.

This is the solution that i'm currently using in my project and its based on offical recipe as can be seen here:
https://github.com/evgenyrodionov/redux-logger#log-only-in-development

I guess predicate option returning false will work but im not sure if that more optimized? Thoughts?

File: app/store/configureStore.js [hunk 1]
```diff
@@ -10,21 +10,28 @@ const logger = createLogger({
   collapsed: false,
   logger: console,
   predicate: (getState, action) => true
-})
+});
```
Comment (mz026) [inline app/store/configureStore.js]: We don't use semi-colons here, please remove them :smile:

File: app/store/configureStore.js [hunk 2]
```diff
@@ -10,21 +10,28 @@ const logger = createLogger({
...
 
-const createStoreWithMiddleware = applyMiddleware(
+let middleware = [
```
Comment (mz026) [inline app/store/configureStore.js]: Let's name this as `middlewares` to be more expressive.
(I know grammatically it's a compound, but adding an `s` make it more readable as an array)

**Comment by mz026** [comment]
@bolicd thanks for pointing out!

IMO, from the performance point of view, initializing the logger middleware only in development mode only is the way to go. On the other hand, by using the `predicate` options we keep the flexibility to do some optional logging in production: like by default we log nothing but we can enable the logger when debugging by setting `window.ENABLE_LOG = true` 

I think your version is better as the optional logging in production might be the minor cases.
I left a review on the code, can you please update it?

**Comment by bolicd** [comment]
@mz026 i've pushed new commit with requested changes. Please review.

Thanks

**Comment by mz026** [comment]
thanks! @bolicd

**Pull request merged**